### ISSUE TITLE
IzzyDroid App not syncthing

### ISSUE THREAD

**Issue by link-leo** [floccusaddon/floccus#1706]
### Which version of floccus are you using?

5.2.6

### How many bookmarks do you have, roughly?

100

### Are you using other means to sync bookmarks in parallel to floccus?

No

### Sync method

Git

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Librewolf

### Describe the Bug

Sync using Librewolf extension to self-hosted Forgejo works great, but bookmarks never load in IzzyDroid app. In fact, allowing sync to run eventually causes phone to overheat.

### Expected Behavior

Bookmarks synced to Android app.

### To Reproduce

Sign into Forgejo repo via Librewolf extension and then sign in via IzzyDroid app.

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
Hi @link-leo 
This is a known issue, I think. Sadly, syncing via git doesn't work on the mobile app yet, due to upstream issues with the mobile framework that floccus uses. This is being tracked here: #1608
Sorry for the disappointment! I'm closing this in favor of the issue above.

**Comment by link-leo** [comment]
Will follow there and continue testing as things change.
